### PR TITLE
Remove GUI handling old CE servers without workspaces 

### PR DESCRIPTION
**Pull request by PeterPetrik**
removed 
 - old account page (pre-workspace) from QML
 - old project page (pre-workspace) from QML (and associated C++ enums)
 - disabled registration for pre-workspace servers
 - fixed tests

### CODE REVIEW

File: app/projectsmodel.cpp [hunk 1]
```diff
@@ -544,17 +543,6 @@ void ProjectsModel::onProjectAttachedToMergin( const QString & )
   listProjectsByName();
 }
 
-void ProjectsModel::onAuthChanged()
-{
-  if ( !mBackend->userAuth() || !mBackend->userAuth()->hasAuthData() ) // user logged out, clear created and shared lists
-  {
-    if ( mModelType == CreatedProjectsModel || mModelType == SharedProjectsModel )
-    {
-      clearProjects();
```
Comment (tomasMizera) [inline app/projectsmodel.cpp:553]: Don't we need to do this for the `WorkspaceProjectsModel`?

Comment (PeterPetrik) [reply app/projectsmodel.cpp:553]: I do not know.... if yes, it is bug in existing code?

**Comment by PeterPetrik** [comment]
linux error is not connected - it is some problem with diff

**Review by tomasMizera — APPROVED** [review]
Looking good 💪

**Pull request merged**